src/controllers/users.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const { UserService } = require('../services/userService');
+const userService = new UserService();
+
+const getAll = (req, res) => {
+  const users = userService.getAll();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+  const foundUser = userService.getUserById(+userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/105124305/203943576-ac08324a-3ccd-42f5-896d-2494cebaeed7.png)
can return error 400 = bad request, if id not valid. (hint isNaN(userId) === true => 400)